scipy/interpolate/interpolate.py

```diff
@@ -1464,6 +1464,33 @@ class RegularGridInterpolator(object):
 
     .. versionadded:: 0.14
 
+    Example
+    -------
+
+    Evaluate a simple example function on the points of a 3D grid:
+
+    >>> from scipy.interpolate import RegularGridInterpolator
+    >>> def f(x,y,z):
+    >>>     return 2 * x**3 + 3 * y**2 - z
+    >>> x = np.linspace(1, 4, 11)
+    >>> y = np.linspace(4, 7, 22)
+    >>> z = np.linspace(7, 9, 33)
+    >>> # Our "data" array is defined by data[i,j,k] = f(x[i], y[j], z[k]):
```

Comment:
Preferably write this as a normal sentence, not as a comment in the interpreter. The code sample should be surrounded by double backticks and there should be blank lines above and below the sentence.